rosdep/base.yaml

```diff
@@ -2596,6 +2603,7 @@ libatomic:
   arch: [gcc-libs]
   debian: [libatomic1]
   fedora: [libatomic]
+  freebsd: [gcc]
```

Comment:
I couldn't figure out if libatomic was provided by gcc

Reply:
I think I'll just remove this rule. It seems that SOME of the gcc versions in FreeBSD have libatomic.so, but not the DEFAULT one.